src/rules/converters/no-unused-variable.ts

```diff
@@ -0,0 +1,20 @@
+import { RuleConverter } from "../converter";
+
+export const NO_UNUSED_VARIABLE_NOTICE =
+    "Please read the following article as the rule behaviour may change on the short term: " +
+    "https://github.com/typescript-eslint/typescript-eslint/issues/1856";
+
+export const convertNoUnusedVariable: RuleConverter = () => {
+    return {
+        rules: [
+            {
+                ruleName: "@typescript-eslint/no-unused-vars",
+                notices: [NO_UNUSED_VARIABLE_NOTICE],
+            },
+            {
+                ruleName: "no-unused-vars",
+                ruleArguments: ["off"],
+            },
+        ],
```

Comment:
🤔 I don't think this is necessary. Other `@typescript-eslint/` rules that override existing ESLint rules don't provide this. Can you remove it?